.Rprofile

```diff
@@ -0,0 +1,37 @@
+# R Configuration for teal.code development
```

Comment:
I think this enforces too many decisions on the developers and we shouldn't include `.Rprofile` on the repos.

This is unprecedented on the organization and tidyverse.

In pharmaverse it appears on `stdm.oak`, but otherwise it's not used anywhere.